pkg/graphdb/conn_sqlite3_unix.go

```diff
@@ -3,5 +3,5 @@
 package graphdb
 
 import (
-	_ "code.google.com/p/gosqlite/sqlite3" // registers sqlite
+	_ "github.com/github.com/mattn/go-sqlite3" // registers sqlite
```

Comment:
ah i was too slow to fix and you saw it :ashamed:

On Thu, Sep 3, 2015 at 5:40 PM, Tianon Gravi <notifications@github.com>
wrote:

> In pkg/graphdb/conn_sqlite3_unix.go
> <https://github.com/docker/docker/pull/16054#discussion_r38712452>:
>
> > @@ -3,5 +3,5 @@
> >  package graphdb
> >
> >  import (
> > -	_ "code.google.com/p/gosqlite/sqlite3" // registers sqlite
> > +	_ "github.com/github.com/mattn/go-sqlite3" // registers sqlite
>
> github.com/github.com
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/docker/docker/pull/16054/files#r38712452>.
>